content/posts/2015-02-02_february-newsletter.md

```diff
@@ -0,0 +1,130 @@
+Title: February Newsletter
+Date: 2015-02-02
+Tags: newsletter
+Slug: february-newsletter-2015
+Author: David Fischer
+
+
+Howdy,
+
+If there are articles, jobs or events you'd like to see in the next newsletter
+or if you have any feedback on San Diego Python, feel free to
+[share with us][].
+
+[share with us]: mailto:sandiegopython-organizers@googlegroups.com
+
+
+Upcoming events
+---------------
+
+[Saturday study group][saturday-meetup] (meetup.com) <br />
+San Diego PyLadies and SD Python are continuing the Saturday study group
+at Pangea cafe. The goal of the Saturday study group is to learn Python
+together in a collaborative environment. Over the next few weeks, the group
+will feature some data mining/analysis presentations by Eric Busboom. If
+you're going to the data science workshop, check it out!
+
+[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
+
+
+[Monthly Meetup][] (meetup.com) <br />
```

Comment:
On 02/02/2015 14:56, David Fischer wrote:
>
> In content/posts/2015-02-02_february-newsletter.md 
> <https://github.com/pythonsd/pythonsd.org/pull/65#discussion_r23967211>:
>
> > +
> > +
> > +Upcoming events
> > +---------------
> > +
> > +[Saturday study group][saturday-meetup] (meetup.com) <br />
> > +San Diego PyLadies and SD Python are continuing the Saturday study group
> > +at Pangea cafe. The goal of the Saturday study group is to learn Python
> > +together in a collaborative environment. Over the next few weeks, the group
> > +will feature some data mining/analysis presentations by Eric Busboom. If
> > +you're going to the data science workshop, check it out!
> > +
> > +[saturday-meetup]: http://www.meetup.com/pythonsd/events/219919725/
> > +
> > +
> > +[Monthly Meetup][] (meetup.com) <br />
>
> Would you prefer to give a more raw presentation or a more polished 
> presentation? I completely understand both.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/pythonsd/pythonsd.org/pull/65/files#r23967211>.
>
Let's do March. We already have a bunch going on in Feb :)

-- 
*Carol Willing*
Developer | Willing Consulting
https://willingconsulting.com